### PR TITLE
Song branch

### PR DESCRIPTION
**Pull request by leepcode**
test cases  is  added
add the test cases of util

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/wepy?pullRequest=2655) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/wepy?pullRequest=2655) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/wepy?pullRequest=2655) it.</sub>

File: package.json [hunk 1]
```diff
@@ -58,13 +58,15 @@
...
-    "nyc": "^15.0.0",
+    "nyc": "^15.1.0",
+    "power-assert": "^1.6.1",
     "prettier": "^1.18.2",
     "read-pkg": "^4.0.1",
     "rollup": "^0.56.5",
     "rollup-plugin-buble": "^0.19.2",
     "rollup-plugin-replace": "^2.0.0",
     "serve-static": "^1.11.1",
+    "supertest": "^4.0.2",
```
Comment (Gcaufy) [inline package.json:69]: 不应该引用 这个库吧。

File: package.json [hunk 2]
```diff
@@ -58,13 +58,15 @@
     "less": "^3.9.0",
     "mkpath": "^1.0.0",
     "mocha": "^6.2.3",
-    "nyc": "^15.0.0",
+    "nyc": "^15.1.0",
+    "power-assert": "^1.6.1",
```
Comment (Gcaufy) [inline package.json:62]: 还有这个，有用到吗？

File: packages/core/test/weapp/util/model.test.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
+
+const expect = require('chai').expect;
+import { parseModel } from '../../../weapp/util/model';
+
+describe('parseModel test', () => {
+    it('没有方括号和点时', () => {
```
Comment (Gcaufy) [inline packages/core/test/weapp/util/model.test.js:6]: 这里统一使用英文吧。

**Pull request closed without merging**